sdk/typescript/src/types/transactions.ts

```diff
@@ -193,6 +193,10 @@ export const OwnedObjectRef = object({
   reference: SuiObjectRef,
 });
 export type OwnedObjectRef = Infer<typeof OwnedObjectRef>;
+export const ModifiedObject = object({
```

Comment:
@666lcz I saw some TS error failures, and tried to add blocks here, but I already forgot what's gong on here, any wisdom? thanks!